modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/organization/management/v1/OrganizationManagementSuccessTest.java

```diff
@@ -1125,7 +1135,182 @@ public void testDisableEmailDomainDiscovery() {
         Assert.assertFalse(isEnabled, "Email domain discovery was not successfully disabled.");
     }
 
-    @Test(dependsOnGroups = "organizationDiscoveryPaginationTests")
+    @Test(groups = "organizationMetaAttributesPaginationTests",
+            dependsOnGroups = "organizationDiscoveryPaginationTests")
+    public void testAddMetaAttributesToOrganizations() {
```

Comment:
Lets add a new line after method signature

Reply:
Addressed with [2122b95](https://github.com/wso2/product-is/pull/21054/commits/2122b959d805379690765472e0f1c6ae56087016)